### PR TITLE
[BACKLOG-35956] : Fix Search Bar

### PR DESCRIPTION
**Pull request by addagudi**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java [hunk 1]
```diff
@@ -1141,11 +1152,13 @@ protected void selectPath( Object selectedElement, boolean useCache ) {
 
     } else if ( selectedElement instanceof Directory ) {
       try {
-        fileTableViewer.setInput( FILE_CONTROLLER.getFiles( (File) selectedElement, null, useCache ).stream().sorted(
-            Comparator.comparing( f -> f instanceof Directory, Boolean::compare ).reversed()
-              .thenComparing( Comparator.comparing( f -> ( (File) f ).getName(),
-                String.CASE_INSENSITIVE_ORDER ) ) )
-          .toArray() );
+        String searchString = txtSearch.getText();
```
Comment (ddiroma) [inline plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java:1155]: @addagudi Please see instructions in the README file in https://github.com/pentaho/pentaho-coding-standards to add checkstyle to your IDE. After implementing the checkstyle, please run it against this file to reformat your changes. Thank you.

Comment (addagudi) [reply plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java]: Fixed.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 10s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by ddiroma — APPROVED** [review]

**Pull request merged**